packages/extension-chakra-storefront/src/hooks/use-add-to-cart-modal.js

```diff
@@ -227,62 +134,252 @@ export const AddToCartModal = () => {
+                                            )
+                                        })}
+                                </Box>
+                                <Box
+                                    display={['none', 'none', 'none', 'block']}
+                                    flex="1"
+                                    paddingX={{lg: '4', xl: '8'}}
+                                    paddingY={{base: '4', lg: '0'}}
+                                >
+                                    <Flex justifyContent="space-between" marginBottom="8">
+                                        <Text fontWeight="700">
+                                            {intl.formatMessage(
+                                                {
+                                                    defaultMessage:
+                                                        'Cart Subtotal ({itemAccumulatedCount} item)',
+                                                    id: 'add_to_cart_modal.label.cart_subtotal'
+                                                },
+                                                {itemAccumulatedCount: totalItems}
+                                            )}
+                                        </Text>
+                                        <Text alignSelf="flex-end" fontWeight="600">
+                                            {productSubTotal &&
+                                                intl.formatNumber(productSubTotal, {
+                                                    style: 'currency',
+                                                    currency: currency
+                                                })}
+                                        </Text>
+                                    </Flex>
+                                    <Stack gap="4">
+                                        <Button as={Link} to="/cart" width="100%" variant="solid">
+                                            {intl.formatMessage({
+                                                defaultMessage: 'View Cart',
+                                                id: 'add_to_cart_modal.link.view_cart'
+                                            })}
+                                        </Button>
+
+                                        <Button
+                                            as={Link}
+                                            to="/checkout"
+                                            width="100%"
+                                            variant="outline"
+                                            rightIcon={<LockIcon />}
+                                        >
+                                            {intl.formatMessage({
+                                                defaultMessage: 'Proceed to Checkout',
+                                                id: 'add_to_cart_modal.link.checkout'
+                                            })}
+                                        </Button>
```

Comment:
We need to add Icons as children:

```suggestion
                                            })}
                                            <LockIcon />
                                        </Button>
```